sections/oxygen_pre_deployment.md

```diff
@@ -39,173 +39,107 @@ When applying the tape, be sure that it is not in contact with any other metal p
-Materials: Distilled water, aquarium pump, 1L and small volume beakers, stirrer and magnetic stirrer, BOD bottles, Winkler reagents, sodium sulfite solution, pipettes and tips, barometer, termometer.
-
-Type *Get All* command for saving the initial sensor configuration to be able to restore old values in case something goes wrong.
-
-1. With the sensor submerged in  the 100 % water, connect to it and start measuring.  Set the Interval property to 30 seconds. (This interval is recommended during the calibration to reduce the risk of self heating in the small Set).
-             
-            container Passkey(1000)
-            
-            Set Interval(30)
-            
-            Save
-2. Wait until both the temperature and the phase measurements are stabilized.
-
-3. Store calibration values by typing:
-           
-           Set Passkey(1000)
-            
-           Do CollectCalDataSat
-The Save command is automatically performed.
-
-4. Set the CalDataAPress property to the actual air pressure in hPa at the site:
-           
-           Set Passkey(1000)
-           
-           Set CalDataAPress (..)
-          
-           Save
-5. Take 2 samples for Winkler measurement.
-
-6. Dry the sensor carefully (make sure that the sensing foil is free from air bubbles) and immerse it in 0 % bath (0 % almost always reads correctly and is part of the calibration process). 
-Wait until both the temperature and the phase measurements have stabilized (approx. 3 min).
-
-7. To store calibration values for 0 %:
-           
-           Set Passkey(1000)
-           
-           Do CollectCalDataZero
-The Save command is automatically performed.
-
-8. To accept the new calibration and store the new coefficients in the sensor:
-          
-           Set Passkey(1000)
-           
-           Do Calibrate
-The  Save command is automatically performed.
-
-9. Rinse well with distilled water and dry the sensor foil carefully and check how it works at air. 
-Save the data to check the sensor performance through time (a barometer is needed). 
-Value of oxygen saturation should be 100 % or higher.
-
-10. Put the sensor back into the 0 % water, the reading should drop to zero.
-
-11. Set back the Interval property to 1 second (or the desired sampling frequency).
-            
-            Set Passkey(1000)
-            
-            Set Interval(1)
-            
-            Save
-12. Type Get All command for saving the final sensor configuration for reference.
-
 ### In situ intercomparison in the tank during ballasting
-This is an extra in situ intercomparison to carry out if  access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before) .
+This is an extra in situ intercomparison to carry out if access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before) (reference PLOCAN).
```

Comment:
@tomhull  Please, can you remove the reference to PLOCAN? we haven't received the doc so there is nothing now

Reply:
I removed it.